src/components/Home/Home.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { NavLink } from 'react-router-dom';
+
+import './home.scss';
+
+import fetchCategories, { joinCategory } from '../../redux/API/features/async_thunk';
+
+export const WorkDone = (dataFetched) => {
+  const dispatch = useDispatch();
+  const data = useSelector((state) => state.food.food);
+  const foods = Object.values(data);
+
+  useEffect(() => {
+    dispatch(dataFetched());
+  }, [dataFetched, dispatch]);
+
+  return foods;
+};
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const handleJoin = (e) => {
+    dispatch(joinCategory(e.target.id));
+  };
+  const foods = WorkDone(fetchCategories);
+  return (
+    <ul className="categories">
+      {foods.length <= 1 ? 'Loading...'
+        : foods.map((food) => (
+          <li key={food.category_id} className="category">
+            <img src={food.category_image} alt={food.category_name} />
+            <h2>{food.category_name}</h2>
+            <NavLink to="seemore"><button id={food.category_id} type="button" onClick={handleJoin}>See More</button></NavLink>
+          </li>
+        )) }
+    </ul>
```

Comment:
- [ ] Please model the cards to look exactly like the given design guideline as shown below 👇 
- [ ] Make sure the card background colors are chequered to look like a chess board
- [ ] Kindly include the arrow icon at the top right of each card as shown below
## Design Guideline
![image](https://user-images.githubusercontent.com/46839548/218238516-45d5f490-0501-42b0-b638-2fe15d1d480a.png)

 ## Your Design
![image](https://user-images.githubusercontent.com/46839548/218238457-22b8ad42-70c5-42e3-99c4-08d894e433ab.png)